assets/src/edit-story/components/canvas/carousel/compactIndicator.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+function CompactIndicatorWithRef({ onClick, isActive, ariaLabel }, ref) {
+  return (
+    <Indicator
```

Comment:
Just to confirm: is it intended that the compact indicator is not draggable? Since it's not really visible what's on a Page, I assume so, confirming just in case.

Reply:
Yes. Can't find the discussion on Slack right now though.

cc @pbakaus to confirm

Reply:
yes, compact indicator is not draggable.

Reply:
That's correct.

_Yeah, I would say that compact mode becomes pretty useless for reordering. It's more for navigating around the pages. - Sam_